python_modules/dagster/dagster/_daemon/asset_daemon.py

```diff
@@ -929,6 +929,22 @@ async def _async_process_auto_materialize_tick(
                 log_message="Automation condition daemon caught an error",
             )
 
+    def _add_auto_materialize_asset_evaluations_in_chunks(
+        self,
+        instance: DagsterInstance,
+        evaluation_id: int,
+        evaluations: list[AutomationConditionEvaluationWithRunIds],
+    ):
+        schedule_storage = check.not_none(instance.schedule_storage)
+        if schedule_storage.supports_auto_materialize_asset_evaluations:
+            chunk_size = int(os.getenv("DAGSTER_ASSET_DAEMON_ASSET_EVALUATIONS_CHUNK_SIZE", "50"))
```

Comment:
should the default here be higher than 50? maybe 200? would be nice to feel confident that this would be unlikely to change behavior for people in most cases. e.g. if there were 4000 new evaluations, 80 separate queries seems like a lot